server.js

```diff
@@ -249,9 +249,22 @@ app
           .getJobs()
           .then((jobs) => {
             let filteredJobs = jobs.map((job) => {
+              let date = new Date(job.timestamp);
               return {
                 id: job.id,
                 title: _.get(job.data.details, bookTitle[req.query.queue_name]),
+                userName: job.data.userName ? job.data.userName : "-",
+                timestamp:
+                  date.getUTCFullYear() +
+                  "-" +
+                  date.getUTCMonth() +
```

Comment:
If `getUTCMonth()` returns a single digit and `getUTCDate()` returns a single digit as well, it looks a bit weird as the date is shown as `2023-5-1`. It would be better if it shown as `2023-05-01`.